core/rspamd/conf/multimap.conf

```diff
@@ -9,3 +9,16 @@ IS_LOCAL_DOMAIN_E {
   selector = "from('smtp'):domain";
   map = "http://{{ ADMIN_ADDRESS }}/internal/rspamd/local_domains";
 }
+
+FORBIDDEN_FILE_EXTENSION {
+    type = "filename";
+    filter = "extension";
+    map = [
+    "/etc/rspamd/override.d/forbidden_file_extension.map",
```

Comment:
You refer to `"/etc/rspamd/override.d/forbidden_file_extension.map"`, but this file does not exist.

Since this file `core/rspamd/conf/forbidden_file_extension.map` does not need to be parsed on startup, you can copy it via the Dockerfile or in start.py.

Reply:
Doh; That's 77d770a2